src/main/java/org/frankframework/frankdoc/model/FrankElement.java

```diff
@@ -173,7 +173,7 @@ private void handlePossibleParameters(FrankClass clazz) {
 	private void handlePossibleForwards(FrankClass clazz) {
 		for(String forwardStr: clazz.getAllJavaDocTagsOf(JAVADOC_FORWARD)) {
 			if(StringUtils.isBlank(forwardStr)) {
-				log.warn("FrankElement [{}] has forwards without a name or description", fullName);
+				log.error("FrankElement [{}] has forwards without a name or description", fullName);
```

Comment:
build moet niet falen als alleen description ontbreekt

Reply:
Als de naam en de beschrijving allebei ontbreken, moet de build dan wel falen?

Reply:
Ook bij deze annotatie is de naam verplicht.

Reply:
Ik heb nu een error als er geen naam en geen description is. Ik heb een warning als er alleen een naam is.